framework/src/modules/chain/synchronizer/block_synchronization_mechanism.js

```diff
@@ -133,35 +192,15 @@ class BlockSynchronizationMechanism extends BaseSynchronizer {
 			'Requesting blocks within ID range from peer',
 		);
 
-		const listOfFullBlocks = await this._requestBlocksWithinIDs(
+		await this._requestAndApplyBlocksWithinIDs(
```

Comment:
original function `_requestBlocksWithinIDs` can be move to the fast chain mechanism from base mechanism